error_handler.py

```
@@ -0,0 +1,273 @@
+"""
+Standardized error handling utilities for the Discord bot.
+Provides consistent error handling patterns and logging across all modules.
+"""
+
+import logging
+import sqlite3
+import discord
+import asyncio
+from typing import Optional, Union, Type, Callable, Any
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import

(F401)